splice_machine/src/test/java/com/splicemachine/derby/utils/SpliceStringFunctionsIT.java

```diff
@@ -464,6 +464,77 @@ public void testInitcapFunction() throws Exception {
         }
     }
 
+    @Test
```

Comment:
Can you add a test with useSpark=true and with/without useNativeSpark=true?

Reply:
Added tests which should automatically pick native spark for broadcast and sortmerge join, if eligible.